src/Effect/Monad/Reader/Transformer.agda

```diff
@@ -115,7 +116,22 @@ liftReaderT MRead = record
   ; local = λ f mx → mkReaderT (λ r₂ → local f (runReaderT mx r₂))
   } where open RawMonadReader MRead
 
-open import Data.Product using (_,_)
+open import Data.Product using (_×_; _,_)
+open import Effect.Monad.Writer.Transformer.Base
+
+module _ {MR' : RawMonoid r g} where
```

Comment:
I don't think this should be an implicit argument as there is no chance it will be reconstructed by unification.